### ISSUE TITLE
Any way to change priority list on gaols? (Titan UWU)

### ISSUE THREAD

**Issue by nowa1991** [quisquous/cactbot#1901]

**Comment by panicstevenson** [comment]
The code, as written currently, puts the players in alphabetical order by their in-game name.

However, I don't think it would be too difficult to implement something you could override with an option. Instead of `data.titanGaols.sort()`, we could do something like:
```js
data.titanGaols.sort((player1, player2) => {
  if (Options.GaolOrder && Options.GaolOrder.length >= 1) {
    const order1 = Options.GaolOrder.includes(player1) ? Options.GaolOrder.indexOf(player1) : 99;
    const order2 = Options.GaolOrder.includes(player2) ? Options.GaolOrder.indexOf(player2) : 99;
    if (order1 < order2) {
      return -1;
    }
    if (order2 > order1) {
      return 1;
    }
    return 0;
  }
  return 0;
});
```

Which could be overwritten in the user options with:
```js
Options.GaolOrder = ["Player One'name", "Player Two'name"];
```

Allowing a set of incoming gaols that would have been:
```
Player One'name => Captain Jimmy => Player Two'name
```
To instead be output as:
```
Player One'name => Player Two'name => Captain Jimmy 
```

Basically leaving everyone else in order, so if you only cared about putting `Player Two'name` to the front you could do:
```js
Options.GaolOrder = ["Player Two'name"];
```
Giving you (for the same example):
```
Player Two'name => Player One'name => Captain Jimmy 
```

**Comment by quisquous** [comment]
You might also want to take a look at #181, where there's also some code to do this.

**Comment by quisquous** [comment]
Also, @nowa1991, would you mind commenting on #1904 if you have been in UWU lately and saying if you saw that timeline bug or not?

**Comment by quisquous** [comment]
I'm going to close this now.  Let me know if there's anything else on this issue you need help with.
